### PR TITLE
[htdocs/vendor/js/react] update react & react-dom 

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Updates the react & react-dom found in htdocs/vendor/js/react and restore the react-dom back to package.json because it's used for latest react hooks that's used for the DQT redesign.

#### Testing instructions (if applicable)

1. Checkout PR
2. run `make`
3. View LORIS

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.11.0",
     "react-addons-create-fragment": "^15.6.2",
+    "react-dom": "^16.12.0",
```
Comment (driusan) [inline package.json]: react-dom doesn't match the version of React 2 lines above. Javascript people: does that matter? (@xlecours ? @HenriRabalais ?)

Comment (maltheism) [reply package.json]: @driusan the version doesn't need to match. Also to note the react we use in LORIS is the latest found on npm and similar with the react-dom.

Comment (HenriRabalais) [reply package.json]: As far as I know, they don't have to match.

**Comment by maltheism** [comment]
@HenriRabalais can you review & approve. Thanks for the comment!

**Comment by maltheism** [comment]
Closing PR in favour of https://github.com/aces/Loris/pull/6097

**Pull request closed without merging**